packages/core/addon/components/navi-vis-c3-chart.js

```diff
@@ -140,6 +141,8 @@ export default C3Chart.extend({
   chart: computed('_config', '_chart', function() {
     if (!get(this, '_chart')) {
       let config = get(this, '_config');
+
+      // eslint-disable-next-line ember/no-side-effects
```

Comment:
What is your link supposed to be? It just opens this same page for me.

Reply:
there is a globals c3 comment above the component declaration that should have handled this linting error.

Reply:
I think the issue is with the `no-side-effects` eslint rule. Since we're mutating the `_chart` property in this computed property, it throws a linting error.

Reply:
Right, it's the no side effects thing.  The comment line is suppressing the linting error.

Reply:
Can we make an issue to clean this up?

Reply:
Yes https://github.com/yahoo/navi/issues/334